server/modules/form/api.js

```diff
@@ -46,7 +62,7 @@ export default {
 
 		const emailValue = await getUserEmail(req.cookies.FTSession_s);
 
-		return res.render('form', {
+		const templateData = {
 			title: 'Signup',
```

Comment:
There are a few places in this file where we're hardcoding 'Signup' as a title property. We could read the title from this `templateData` object instead in those places 🙂